roles/lib_utils/action_plugins/sanity_checks.py

```diff
@@ -466,6 +466,34 @@ def validate_json_format_vars(self, hostvars, host):
             raise errors.AnsibleModuleError(msg)
         return None
 
+    def check_for_oreg_password(self, hostvars, host, odt):
+        """Ensure oreg_password is defined when using registry.redhat.io"""
+        reg_to_check = 'registry.redhat.io'
+        err_msg = ("oreg_user and oreg_password must be provided when"
```

Comment:
Should be "oreg_auth_user" and "oreg_auth_password" instead of "oreg_user" and "oreg_password", am I right?